dune/stuff/common/configuration.hh

```diff
@@ -82,16 +91,15 @@ static const bool configuration_log_on_exit = false;
 static const std::string configuration_logfile = "data/log/dsc_parameter.log";
 
 
-template <class T>
+template < class T >
 struct Typer {
-  typedef typename std::conditional<std::is_convertible<T, std::string>::value, std::string, T>::type type;
+  typedef typename std::conditional< std::is_convertible< T, std::string >::value, std::string, T >::type type;
 };
 
 } // namespace internal
 
 
-class Configuration
-  : public Dune::ParameterTree
+class Configuration : public Dune::ParameterTree
```

Comment:
I was more worried about situations like [here](https://github.com/wwu-numerik/dune-stuff/blob/master/dune/stuff/functions/checkerboard.hh#L28). In general, that is handled well, although I do not like the additional indentation there. Will investigate...